### PR TITLE
rmarkdown tutorial - initial commit

### PR DESCRIPTION
**Pull request by richardjtelford**
Initial draft of R markdown tutorial.

### CODE REVIEW

File: Rmarkdown/Rmarkdown-intro.Rmd [hunk 1]
```diff
@@ -0,0 +1,643 @@
...
+
+````
+```{r import, echo = FALSE}`r ''`
+data(penguins, package = "palmerpenguins)
+```
+````
+
+It starts with three back-ticks, followed by braces.
+Inside the braces, the "r" indicates that this is a chunk in the R language.
+The next word in the optional chunk name.
```
Comment (audhalbritter) [inline Rmarkdown/Rmarkdown-intro.Rmd:171]: Good practice to give meaningful chunk names

File: Rmarkdown/Rmarkdown-intro.Rmd [hunk 2]
```diff
@@ -0,0 +1,643 @@
...
+```
+
+### Citations
+
+Use the citation keys from the bibliography to cite the reference in the text.
+So to cite the reference with the citation key "Cooper2017" we would use.
+
+
+    Reproducible code [@Cooper2017] ...
+    @Cooper2017 discuss ...
```
Comment (audhalbritter) [inline Rmarkdown/Rmarkdown-intro.Rmd:440]: Is [-@Cooper2017] doing the same thing?

File: Rmarkdown/Rmarkdown-intro.Rmd [hunk 3]
```diff
@@ -0,0 +1,643 @@
...
+penguins %>% 
+  #rename some columns
+  rename(length_mm = bill_length_mm, depth_mm = bill_depth_mm) %>% 
+  #get first five rows
+  slice(1:5) %>% 
+kbl(booktabs = TRUE, caption = "The top penguins") %>% 
+  kable_styling(full_width = TRUE) %>%
+  add_header_above(c(" ", " ", "bill" = 2, "body" = 2, " ", " "))
+  
+```
```
Comment (audhalbritter) [inline Rmarkdown/Rmarkdown-intro.Rmd:339]: Could add the code to make a table landscape (wide table)

File: Rmarkdown/Rmarkdown-intro.Rmd (hunk 2 above)
Comment (richardjtelford) [reply Rmarkdown/Rmarkdown-intro.Rmd]: no. [-@Cooper2017]  will render as (2017). Useful occasionally. When I started I didn't realise I could do @Cooper2017 so would write Cooper et al [-@Cooper2017]

File: Rmarkdown/Rmarkdown-intro.Rmd (hunk 3 above)
Comment (richardjtelford) [reply Rmarkdown/Rmarkdown-intro.Rmd]: That is a latex package to addon - now link to the solution

**Pull request merged**